dashboard/app/views/videos/embed.html.haml

```diff
@@ -1,6 +1,14 @@
+/ This view creates a page that transparently provides fallback-player support to the specified YouTube video.
+/ See VideosController#embed for usage
+!!! 5
+- video = local_assigns[:video]
+- process_fallback_videos = current_user.try(:admin?) && !(Rails.env.production? || Rails.env.test?)
```

Comment:
The conditional is a little tricky to understand.  I think the following is equivalent, but I don't quite understand the intent. I see why you wouldn't want to process fallback videos in test, but it seems like we *do* want to process fallback videos in production. (?)

process_fallback_videos = current_user.try(:admin?) && !Rails.env.production? && !Rails.env.test?